libs/backend/src/models/game/game-config.ts

```diff
@@ -0,0 +1,21 @@
+import { Schema } from "mongoose";
+import { GameVisibilityEnum } from "types";
+
+/**
+ * IDEA: Eventually add gameOptions types
+ * there could be different gameOptionss, hidden generated by the system, and public testcases
+ * this could mitigate users that just look at the public test cases and thus help catch code that technically is wrong, but passes the visible gameOptionss (to all users).
+ */
+const gameOptionsSchema = new Schema({
+	mexGameDurationInSeconds: {
```

Comment:
```suggestion
	maxGameDurationInSeconds: {
```
follow a type course

![omegalul](https://cdn.7tv.app/emote/6042089e77137b000de9e669/4x.webp)